.changelog/16000.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:breaking-change
+connect: Fix configuration merging for transparent proxy upstreams. Proxy-defaults and service-defaults config entries were not correctly merged for implicit upstreams in transparent proxy mode and would result in some configuration not being applied. To avoid issues when upgrading, ensure that any proxy-defaults or service-defaults have the correct information for upstreams, as this configuration will become active.
+```
```

Comment:
> To avoid issues when upgrading, ensure that any proxy-defaults or service-defaults have the correct information for upstreams, as this configuration will become active.

Is this part referencing the wildcard upstream config that you're ensuring gets created? I guess I'm not sure what "as this configuration will become active" means so just wondering if we can reword a bit?